source/help/getting-started/configuring-notifications.rst

```diff
@@ -67,6 +67,9 @@ keywords <https://docs.mattermost.com/help/settings/account-settings.html#words-
    Notifications**. By default, all channels use the global setting
    configured in *Account Settings*.
    
+.. note::
```

Comment:
Is a note styling necessary for this? Do we have standards for when we use notes and other styling vs text?

This is only for desktop app though right? We should specify in the first few words and maybe link to the download page. 

Also specify which Mattermost icon are we talking about?